_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -320,7 +320,7 @@
 "agh" = (/obj/structure/mopbucket,/obj/item/reagent_containers/glass/bucket,/obj/item/mop,/turf/open/floor/wood/f13/old,/area/f13/sunny_dale)
 "agi" = (/turf/closed/wall/f13wood{icon_state = "girder-b"; dir = 1},/area/f13/den)
 "agj" = (/obj/structure/fence/corner{icon_state = "corner"; dir = 9},/turf/open/floor/plating/f13/outside/road{icon_state = "outerpavement"; dir = 4},/area/f13/den)
-"agk" = (/obj/structure/window/fulltile/wood_window,/turf/open/space/basic,/area/f13/den)
+"agk" = (/turf/closed/indestructible/rock,/obj/machinery/door/unpowered/wooddoor,/obj/structure/barricade/wooden/crude,/turf/open/floor/plating/f13/inside/mountain,/area/f13/underground/mountain)

```

Comment:
Feel free to discard my map changes, they're just supposed to be a single space square in the Den converted to normal flooring.